src/core/reducer.ts

```diff
@@ -97,6 +97,11 @@ function initializeDeltalog(
   action: ActionShape.MakeMove | ActionShape.Undo | ActionShape.Redo,
   move?: Move
 ): State {
+  if (state.plugins.log) {
+    const pluginLogMetadata = state.plugins.log.data.metadata;
+    action.payload['metadata'] = pluginLogMetadata;
+  }
+
```

Comment:
If we move `metadata` up to the top-level of log entries and enable the metadata plugin by default, we can move this block after `logEntry` is declared and make it:

```ts
const pluginLogMetadata = state.plugins.log.data.metadata;
if (pluginLogMetadata) {
  logEntry.metadata = pluginLogMetadata;
}
```

Reply:
Good idea! Fixed.